src/services/herolist.js

```diff
@@ -0,0 +1,68 @@
+const heroList = [{

```

Comment:
I think this is the same problem mentioned by the mentor. List implies the what we will work with at it might change. I would use a different name.

Reply:
that's the mock hero database, and is literally a list